webapp/lib/lazy_list_view_model.dart

```diff
@@ -80,18 +89,38 @@ class LazyListViewModel {
       // If the scroll area width changed, then recalculate the scroll length
       // because the item heights and thus the scroll length depends upon the scroll area width.
       _scrollLengthRecalc ??= new Future.delayed(const Duration(seconds: 2), () {
-        _scrollLength = _listView.children.fold(0, (len, element) => len + element.clientHeight);
+        _scrollLength = _listView.children.fold(0, (len, element)
+            => element == _scrollPad ? _scollPadMinHeight : len + element.clientHeight);
```

Comment:
instead of the _scrollPadMinHeight, shouldn't this be the actual height of the _scrollPad, potentially after the other elements have been resized?

Reply:
No, the _scrollLength is the vertical distance of the DOM elements that have been realized but does not include the items in the list that are not in the DOM. The `_scrollPadMinHeight` ensures a small blank space at the end of the list so that the users knows they have reached the end of the list.

Reply:
that makes sense now, thanks @danrubel !